src/test/java/gift/ExceptionServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package gift;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(SpringExtension.class)
+public class ExceptionServiceTest {
+    private ExceptionService exceptionService;
+    private String validName;
+    private String longName;
+    private String invalidSpecialCharName;
+    private String kakaoName;
+
+    @BeforeEach
+    public void setup() {
+        exceptionService = new ExceptionService();
+        validName = "ValidName";
+        longName = "ThisNameIsWayTooLong";
+        invalidSpecialCharName = "@@이름name";
+        kakaoName = "카카오Name";
+    }
+
+    @DisplayName("정상 동작 테스트")
+    @Test
+    public void testFindNameException_ValidName() {
+        // No exception should be thrown
+        exceptionService.findNameException(validName);
+    }
```

Comment:
예외를 throw하지 않는걸 테스트하고 싶으시군요! 
테스트에 junit을 사용하셨는데, 다음과 같은 함수를 사용하시면 예외가 발생하지 않는 상황을 테스트해보실 수 있습니다.
```suggestion
    @DisplayName("정상 동작 테스트")
    @Test
    public void testFindNameException_ValidName() {
        assertDoesNotThrow(() -> {
            exceptionService.findNameException(validName);
        });
    }
```